stdlib/public/Concurrency/Task.swift

```diff
@@ -76,6 +76,45 @@ import Swift
 /// like the reason for cancellation.
 /// This reflects the fact that a task can be canceled for many reasons,
 /// and additional reasons can accrue during the cancellation process.
+///
+/// ### Task closure lifetime
+/// Tasks initialized with a closure, representing the code that will be executed by this task.
+///
+/// Once this code has ran to completion, and the task therefore has completed, resulting in either
```

Comment:
"run to completion"

Reply:
Apple style guide uses "once" to mean "one time" and "after" for time ordering:

```suggestion
/// After this code has run to completion, the task has completed, resulting in either
```